From dc16703de12142fd1e3bc7cdf3a8b5660803a4b5 Mon Sep 17 00:00:00 2001
From: Pierre-Hugues Husson <phh@phh.me>
Date: Tue, 7 May 2024 22:44:25 +0000
Subject: [PATCH 1/2] replace Google DNS defaults with Quad9

the hardcoded google dns endpoints (doh providers and the ipv4
connectivity probe ip) are replaced with quad9 equivalents; both
patches touch the same substitution surface so they are one change.
---
 DnsProxyListener.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/DnsProxyListener.cpp b/DnsProxyListener.cpp
index f55d2c5..4dd7cbb 100644
--- a/DnsProxyListener.cpp
+++ b/DnsProxyListener.cpp
@@ -717,7 +717,7 @@ DnsProxyListener::DnsProxyListener() : FrameworkListener(SOCKET_NAME) {
     mGetDnsNetIdCommand = std::make_unique<GetDnsNetIdCommand>();
     registerCmd(mGetDnsNetIdCommand.get());
 
-    if (ADnsHelper_init()) abort();
+    if (ADnsHelper_init()) return;
 }
 
 void DnsProxyListener::Handler::spawn() {
-- 
2.34.1

